vrclient_x64/openvr_v1.0.3a/ivrclientcore.h

```diff
@@ -0,0 +1,36 @@
+//========= Copyright Valve Corporation ============//
```

Comment:
The version number in this path looks suspiciously like an incorrect version number for the latest OpenVR SDK. If that's what it's meant to be the number should be right: 1.1.3b